CHANGELOG.md

```diff
@@ -11,10 +11,12 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Show an ellipsis if 10 cases or more to display with loqusdb matches
 - Apply default gene panel on return to cancer variantS from variant view
 - A new blog post for version 4.17
+- Tooltip to better describe Tumor and Normal columns in cancer variants
 
 ### Fixed
-### Changed
 
+### Changed
+- Turn tumor and normal allele frequencies to decimal numbers in tumor variants page
```

Comment:
I'm being picky here. These values are not frequencies, but fractions 😁 

```suggestion
- Turn tumor and normal allelic fraction to decimal numbers in tumor variants page
```

Reply:
That's why I asked you!